document/js-api/index.bs

```diff
@@ -1150,7 +1150,7 @@ WebAssembly Module Records have the following methods:
 1. Let |importsObject| be a new Object.
 1. For each |key| →  |value| of |imports|,
     1. Let |moduleImportsObject| be a new Object.
-    1. For each |importedName| →  |importedValue| of |imports|,
+    1. For each |importedName| →  |importedValue| of |value|,
```

Comment:
Correct the double space after the arrow while here? Although it's also done above... copypasta?

Reply:
I'll fix these in a follow-on patch.